src/utils/Cypher.ts

```diff
@@ -136,4 +132,17 @@ export default class Cypher {
       }
     }
   }
+
+  private static async getCryptoSetting(tenantID: string): Promise<CryptoSetting> {
+    const cryptoSettings = (await SettingStorage.getCryptoSettings(tenantID)).crypto;
+    if (!cryptoSettings) {
+      throw new BackendError({
+        source: Constants.CENTRAL_SERVER,
+        module: MODULE_NAME,
+        method: 'getCryptoSetting',
+        message: 'No Crypto Settings found in the database'
```

Comment:
Please use the tenant ID information to build a more informative error message with the tenant name and/or its ID.